src/graphql/utilities/build_client_schema.py

```diff
@@ -175,6 +175,7 @@ def build_enum_def(enum_introspection: Dict) -> GraphQLEnumType:
             description=enum_introspection.get("description"),
             values={
                 value_introspect["name"]: GraphQLEnumValue(
+                    value=value_introspect.get("name"),
```

Comment:
You can change this to value_introspect["name"] since the name must be always set and we do this for the key as well.

Reply:
D'oh, good point, done!